printer_mods/Reiten966/Adafruit_Neopixel_Sidebars/.cfg/Print_Progress_Sidebar.py

```diff
@@ -0,0 +1,52 @@
+[neopixel side_bars]
+pin: expander:PB1 # replace with your own pin
+chain_count: 32
+color_order: GRBW
+initial_RED: 0.6
+initial_GREEN: 0.0
+initial_BLUE: 0.0
+initial_WHITE: 0.0
+
+[gcode_macro M73]
+#   store progress % and remaining time as variables
+rename_existing: M733
+variable_p: 0.0
+variable_r: 0.0
+variable_leds: 32 #leds count that I want to use as progressbar
+variable_offset: 1 #index of the first led of the progressbar at the whole long neopixel strip. begins with 1
+default_parameter_P: 0.0
```

Comment:
default_parameter is deprecated now. See https://docs.mainsail.xyz/faq/klipper_warnings/deprecated_option#removal-of-default_parameter_-for-g-code-macros for more info